pkg/operator/ceohelpers/qourum_check.go

```diff
@@ -26,11 +26,11 @@ func (c *AlwaysSafeQuorumChecker) IsSafeToUpdateRevision() (bool, error) {
 
 // QuorumCheck is just a convenience struct around bootstrap.go
 type QuorumCheck struct {
-	configMapLister corev1listers.ConfigMapLister
-	namespaceLister corev1listers.NamespaceLister
-	infraLister     configv1listers.InfrastructureLister
-	operatorClient  v1helpers.StaticPodOperatorClient
-	etcdClient      etcdcli.AllMemberLister
+	configMapLister    corev1listers.ConfigMapLister
+	namespaceLister    corev1listers.NamespaceLister
+	infraLister        configv1listers.InfrastructureLister
+	operatorClient     v1helpers.StaticPodOperatorClient
+	etcdClient         etcdcli.AllMemberLister
```

Comment:
that's an odd formatting, tab vs. spaces maybe?

Reply:
make update-gofmt  done